examples/simple-chatbot/client/react-native/src/hooks/useVoiceClientNavigation.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useNavigation } from '@react-navigation/native';
+import { useVoiceClient } from '../context/VoiceClientContext';
+
+export const useVoiceClientNavigation = () => {
+  const navigation = useNavigation();
+  const { inCall } = useVoiceClient();
+
+  useEffect(() => {
+    if (inCall) {
+      // TODO, refactor this
```

Comment:
Is there still something you wanted TODO here before merging?

Reply:
Yep, I have just fixed the typescript warning. Thank you.